airbyte-cdk/python/unit_tests/sources/declarative/requesters/paginators/test_request_option.py

```diff
@@ -9,15 +9,9 @@
 @pytest.mark.parametrize(
     "test_name, option_type, field_name, should_raise",
     [
-        ("test_limit_path_no_field_name", RequestOptionType.path, None, False),
```

Comment:
Looks like none of these `should_raise` anymore so the test body can be updated.

That said, it's not immediately obvious to me why the non-`.path` cases are being deleted.

Reply:
ah yes good point i'll clean this test up